dojo-dungeon/src/dojo/rooms/rooms/riddle/Fragen_Pattern.java

```diff
@@ -126,7 +117,7 @@ private BiConsumer<Task, Set<TaskContent>> showAnswersOnHud() {
   private Quiz newFreeText() {
     String questionText =
         "Welches Design-Pattern wird in dem UML-Klassendiagramm unter \""
-            + FILENAME
+            + "dojo-dungeon/todo-assets/Fragen_Pattern/UML_Klassendiagramm"
```

Comment:
bitte keine magic-strings hier. die konstante war sinnvoll. bitte zurückändern.

Reply:
in 124daa4